### DISCUSSION TITLE
discord/matrix/irc for the project?

### DISCUSSION THREAD

**Discussion by SomeGuyNamedMay** [nix-community/stylix#82] [Ideas]
it would be nice for having more casual conversations about the project in my opinion.

**Comment by danth** [comment]
Matrix is probably the best choice given that there are Matrix rooms for NixOS and Home Manager already.

**Reply by SomeGuyNamedMay** [reply]
i have no clue how setting up social networking stuff for projects works but when/if you do it would be a good idea to put that in the readme

**Reply by danth** [reply]
Created a room at https://matrix.to/#/#stylix:danth.me
